src/main/java/Menu.java

```diff
@@ -60,6 +62,23 @@ public static int showPrincipalMenuAndRetrieveOption(Scanner sc) {
         }
         return selectedOption;
     }
+    private static void createTeachers() {
+        for (int i = 0; i < numberOfTeacher; i++) {
```

Comment:
We could use the getValidName() method  
teacherName = getValidName("teacher"+i)